ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts

```diff
@@ -3,14 +3,13 @@ import { StudyLanguage } from '../models/studyLanguage';
 import { ConfigurationService } from '../services/configuration.service';
 import { LanguageService } from '../services/languageService.service';
 import { LanguageServiceAgent } from '../services/serviceAgents/languageServiceAgent.service';
-import { isNullOrUndefined } from 'util';
```

Comment:
I decided to remove `isNullOrUndefined` and use native code instead. I've got a warning here: 
```
WARNING in C:\epam\ddp-angular\ddp-workspace\projects\ddp-sdk\src\lib\services\languageService.service.ts depends on 'util'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```